chpl/chpl-service/src/main/java/gov/healthit/chpl/permissions/domains/ActionPermissions.java

```diff
@@ -18,6 +19,9 @@
     @Autowired
     private CertifiedProductDAO certifiedProductDAO;
 
+    @Autowired
+    private CertifiedProductSearchDAO certifiedProductSearchDAO;
```

Comment:
Is this used? Looks like it was added but I don't see it used in the rest of the changes git has shown in the class.